module/core/src/Infrastructure/Serializer/SerializerInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Ergonode\Core\Infrastructure\Serializer;
```

Comment:
I think it should been in  `SharedKernel` module

Reply:
I think this is not shared functionality other application can use own serializer matching needed requirements